temba/contacts/models.py

```diff
@@ -1852,7 +1852,7 @@ def raw_tel(self):
         if tel:
             return tel.path
 
-    def send(self, text, user, trigger_send=True, response_to=None, message_context=None, connection=None,
+    def send(self, text, user, trigger_send=True, response_to=None, message_context=None, connection=None, metadata=None,
```

Comment:
`metadata` shouldn't be a parameter here - a method like this should take things which end up in metadata, so param here should be `quick_replies`